backend/tests/index.test.ts

```diff
@@ -14,13 +14,16 @@ afterAll(async () => {
 });
 
 describe('Server startup', () => {
-  it('should start the server in non-production mode', (done) => {
+  it('should start the server in non-production mode', async () => {
     process.env.NODE_ENV = 'development';
```

Comment:
If you leave this line in then the test would use the port defined in .env right? Is that the intent?